pytorch/training/docker/1.8/py3/Dockerfile.cpu

```diff
@@ -78,6 +78,27 @@ RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
 COPY changehostname.c /
 COPY start_with_right_hostname.sh /usr/local/bin/start_with_right_hostname.sh
 
+# Install OpenSSH for MPI to communicate between containers, allow OpenSSH to talk to containers without asking for confirmation
+RUN apt-get update \
+ && apt-get install -y  --allow-downgrades --allow-change-held-packages --no-install-recommends \
```

Comment:
nit: is this line required? Seems redundant

Reply:
I directly use the install commands in GPU training dockerfile [here](https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.8/py3/cu111/Dockerfile.gpu#L287-L305). prefer to leave it as is to avoid unexpected behaviors.